src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_params.py

```diff
@@ -273,3 +277,15 @@ def load_arguments_sb(self, _):
         c.argument('alias', options_list=['--alias', '-a'], id_part=None, help='Name of Geo-Disaster Recovery Configuration Alias')
         c.argument('namespace_name', options_list=['--namespace-name'], id_part=None, help='Name of Namespace')
         c.argument('authorization_rule_name', arg_type=name_type, help='Name of Namespace AuthorizationRule')
+
+    # Standard to Premium Migration: Region
+
+    with self.argument_context('servicebus migration start') as c:
+        c.argument('namespace_name', arg_type=name_type, help='Name of Standard Namespace')
+        c.argument('target_namespace', options_list=['--target-namespace'], validator=validate_target_namespace, help='Name (if within the same resource group) or ARM Id of Premium Service Bus namespace name, which is part of migration')
+        c.argument('post_migration_name', options_list=['--post-migration-name'],
+                   help='Name of Standard Namespace post migration')
```

Comment:
IMO it would make more sense for this property to be set in the "complete" operation if that's possible before you GA, but given this setup, you just need more clarity.  I would recommend something like: "The alias assigned to the Standard namespace when the migration is complete." I would then add a long summary: "Since the connection string will be updated to point to the Premium namespace, use this name to refer back to the Standard namespace."

Reply:
Post migration name : 

Standard to premium migration feature does not migrate the data from the Standard Namespace to the Premium namespace. Given this, The recommended workflow to migrate an application from Standard to Premium, without incurring delays in message processing is as follows

1.	Start-Migration. Which starts replicating the nested resources from Standard namespace to Premium Namespace  AND creates the DNS record with the Post Migration name.
2.	User creates a listener application with the Post Migration namespace name, to drain the messages post migration.
3.	Complete-Migration. Which will repoint the standard namespace DNS to the Premium namespace at which point all the Send and Receive operations to the standard namespace are routed to the Premium namespace.

To enable this workflow, the post-migration DNS is created as part of Start-Migration.